### ISSUE TITLE
Monthly Leaderboard

### ISSUE THREAD

**Issue by kevinyhong** [CiaranSanders/unthinking-majority-website#1]
Would be good to see a monthly leaderboard or some other showcase-like thing that pulls from some of the discord channels that get in-game posts (like #lookwhatifound or #dead4head). Could also be used to help compile the highlights of the Month In Review.

**Comment by ciaransanders** [comment]
Yeah that could definitely be an awesome addition for the site. One of the things we tried keeping in mind with the website is that discord is first and foremost the main hub of the clan. So with that in mind it's always good to come up with ways that the site interacts with discord more.

Currently we use webhooks to simply post new PB submissions from the website. But I think for scanning channels and relaying said information back to the site we might be getting into discord bot territory. Building a bot has been brought up before, and I think it would be very cool to have. Like you said, having leaderboards for #lookwhatifound or #dead4head channels could be fun; might be a logistical challenge at first. Could also re-implement the old system of submitting new personal bests, just through a designated discord channel which then gets relayed to the website.

I like the idea, there's just a lot of other logistics we'd have to keep working on to get to a point where we can have a bot.

**Comment by kevinyhong** [comment]
Got another developer mod now lol
